source/__tests__/userFlow.test.js

```diff
@@ -0,0 +1,332 @@
+describe('Basic user flow for Website', () => {
+
+    beforeAll(async () => {
+        await page.goto('https://digitaldr3amt3am-journal.netlify.app');
+    });
+
+    /**
+     * Converts a date string from "Month Day, Year" format to "YearMonthDay" format
+     * @param {string} dateString - Date string in "Month Day, Year" format
+     * @returns {string} - Date string in "YearMonthDay" format (e.g., "May 24, 2024" -> "20240524")
+     */
+    const convertDateString = (dateString) => {
```

Comment:
This looks the same as the `formatDateToYYYYMMDD` function in the `date.util.js` file, could you import that and use it instead?